index.bs

```diff
@@ -797,16 +763,44 @@ The <code>length</code> property of the <code>cancel</code> method is <b>1</b>.
   <li> Otherwise, return <b>undefined</b>.
 </ol>
 
+<h4 id="cancel-readable-stream">CancelReadableStream ( stream )</h4>
+
+<ol>
+  <li> If <b>this</b>@\[[state]] is <code>"closed"</code> or <code>"errored"</code>, return
+    <b>this</b>@\[[closedPromise]].
+  <li> Let <b>this</b>@\[[queue]] be a new empty List.
+  <li> Call-with-rethrow CloseReadableStream(<var>stream</var>).
+  <li> Let <var>sourceCancelPromise</var> be PromiseInvokeOrNoop(<b>this</b>@\[[underlyingSource]],
+    <code>"cancel"</code>, (<var>reason</var>)).
```

Comment:
It looks like I missed a spot in 14bbebcfa9f0e4d8776d33f7b26342124f6bec37; should be «‍_reason_» instead of (_reason_).